### PR TITLE
ADD first version of README

### PR DESCRIPTION
**Pull request by pjheden**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Add additional notes about how to deploy this on a live system
+
+## Contributing
+
+Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Versioning
+
+We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/your/project/tags).
```
Comment (jkimblad) [inline README.md:36]: Link should be https://github.com/jkimblad/jabberjay/tags

File: README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Versioning
+
+We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/your/project/tags). 
+
+## Authors
+
+* **Jakob Kimblad** - *Initial work* - [jkimblad](https://github.com/jkimblad)
```
Comment (jkimblad) [inline README.md:40]: "Jakob"?!??? Jacob, tyvm

File: README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* **Julian Hedén** - *Initial work* - [pjheden](https://github.com/pjheden)
+
+See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project.
```
Comment (jkimblad) [inline README.md:43]: Link should be https://github.com/jkimblad/jabberjay/contributors

**Review by jkimblad — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (pjheden) [reply README.md:40]: aj då, sorry!

**Review by jkimblad — APPROVED** [review]
LGTM

**Pull request merged**